tensorflow/contrib/cmake/external/gemmlowp.cmake

```diff
@@ -14,8 +14,8 @@
 # ==============================================================================
 include (ExternalProject)
 
-set(gemmlowp_URL https://github.com/google/gemmlowp/archive/6a2a90822e8546fc2bfa7044de0faf1c1cb4862f.zip)
-set(gemmlowp_HASH SHA256=3447948d219f3270383766bbe08942888c0eb4e0ca6663c0e0548502ec5bb77d)
+set(gemmlowp_URL https://github.com/google/gemmlowp/archive/d4d1e29a62192d8defdc057b913ef36ca582ac98.zip)
```

Comment:
Should we keep this consistent with https://github.com/tensorflow/tensorflow/blob/master/tensorflow/workspace.bzl#L182

Reply:
Sure I will update to the new URL and HASH. The error looks like this: internal/platform.h:28:10: fatal error: 'malloc.h' file not found. This is because on Mac, malloc.h is not directly under /usr/include directory.
The [malloc.h](https://github.com/google/gemmlowp/blob/6a2a90822e8546fc2bfa7044de0faf1c1cb4862f/internal/platform.h#L28) should be declared by stdlib.h (See [malloc](https://linux.die.net/man/3/malloc) man page].